### PR TITLE
Review documentation (#13)

### PR DESCRIPTION
**Pull request by maptuhec**
* Update aeproject documentation

* Fix comments for documentation

* Small changes of the document

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,9 +27,14 @@ the contracts, tests and deployment files and scripts. Docker configuration
...
 
 ## Compile sophia contracts
+```
+aeproject compile
+```
 
-The **compile** command compiles sophia contract. Files should be with .aes
-file extension. Default directory is $projectDir/contracts.
+The **compile** command compiles Sophia contract. Files should be with .aes
+file extension. Default directory is $projectDir/contracts. The result of the compilation is the contract bytecode
```
Comment (vsavovlime) [inline README.md]: It's not necessary to be with .aes

Comment (maptuhec) [reply README.md]: Are we sure that the compilation works with other file extension?

Comment (vsavovlime) [reply README.md]: Yes

Comment (vsavovlime) [reply README.md]: You might write something: "it's recommended to use .aes extension"

**Pull request closed without merging**